Makefile

```diff
@@ -1,11 +1,21 @@
+install-poetry:
+	curl -sSL https://install.python-poetry.org | python3 -
 
+install-homebrew:
```

Comment:
should `install-homebrew`, `install-pyenv`, and `install-python-3.8` all be called from the setup script if they're not available? Or at least add it to the README that you can use those commands if needed.

Also, if we're giving all these tools for setup, do we still need to downgrade to python 3.8?

Reply:
Let's downgrade to 3.8 for now - and I'll add these commands as optional to the Readme.